plugins/python_3_script/help.md

```diff
@@ -134,6 +134,7 @@ If installation fails, try increasing the `Timeout` connection input to `900` (1
 
 # Version History
 
+* 4.0.7 - Updated the SDK | Add handler to run function as a separate process
```

Comment:
if you are bumping the version of python used in the supported version
`supported_versions: ["Python 3.9.18"]` 

this also may be worth adding to the version to the changelog